sql_generators/active_users/__init__.py

```diff
@@ -49,14 +49,24 @@ def generate(target_project, output_dir, use_cloud_function):
     The parent folders will be created if not existing and existing files will be overwritten.
     """
     env = Environment(loader=FileSystemLoader(str(THIS_PATH / "templates")))
+    output_dir = Path(output_dir) / target_project
+    # query templates
     mobile_query_template = env.get_template("mobile_query.sql")
     desktop_query_template = env.get_template("desktop_query.sql")
     focus_android_query_template = env.get_template("focus_android_query.sql")
-    metadata_template = "metadata.yaml"
-    view_template = env.get_template("view.sql")
+    # view templates
     focus_android_view_template = env.get_template("focus_android_view.sql")
     mobile_view_template = env.get_template("mobile_view.sql")
-    output_dir = Path(output_dir) / target_project
+    view_template = env.get_template("view.sql")
+    # metadata template
+    metadata_template = "metadata.yaml"
+    # checks templates
+    desktop_checks_template = env.get_template("desktop_checks.sql")
+    focus_android_checks_template = env.get_template("focus_android_checks.sql")
+    fenix_checks_template = env.get_template("fenix_checks.sql")
+    focus_ios_checks_template = env.get_template("focus_ios_checks.sql")
+    firefox_ios_checks_template = env.get_template("firefox_ios_checks.sql")
+    klar_ios_checks_template = env.get_template("klar_ios_checks.sql")
 
     for browser in Browsers:
         if browser.name == "firefox_desktop":
```

Comment:
Could we avoid repeating `if browser.name == "firefox_desktop":` if we just places the check render logic inside the same block as the query render call?

Reply:
I had that before but the when translating the different checks I saw that the checks are very specific for each `browser.name`

and yes for firefox_desktop I could move it up but for the rest not so for me it's simpler to read to do the checks in one place all together and not put some here and some there

or am I missreading your point?